src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Label.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import labelStyles from './label.scss';
+
+const TOGGLE_ICON = 'chevron-down';
+
+export default class Label extends React.PureComponent {
+    props: {
+        onClick?: () => void,
+        children: string,
+        icon?: string,
+    };
+
+    button: HTMLElement;
```

Comment:
Even if I don't use things specific to the `HTMLButtonElement`. Should we always use the most general type or the most specific type?